composer.json

```diff
@@ -11,13 +11,15 @@
     }
   ],
   "require": {
-    "php": ">=7.1.0",
-    "nette/forms": "^v2.4.0",
-    "nette/http": "^v2.4.0",
-    "nette/utils": "^v2.4.0",
-    "nette/di": "^v2.4.0",
-    "nette/application": "^v2.4.0",
-    "latte/latte": "^v2.4.0"
+    "php": ">=7.2.0",
+    "ext-json": "*",
+    "latte/latte": "^2.10",
+    "nette/application": "^3.1",
+    "nette/caching": "^3.1",
```

Comment:
nette/caching je tu navic? Minule tu nebylo.

Reply:
Cache se využívá už i v současné verzi - https://github.com/contributte/fileupload/blob/master/src/FileUploadControl.php#L266. Možná by to tak mělo být v závislostech.